R/ranger/bundled/patches/make_lib_and_install.patch

```diff
@@ -0,0 +1,44 @@
+diff --git before/ranger-0.13.1/cpp_version/CMakeLists.txt after/ranger-0.13.1/cpp_version/CMakeLists.txt
```

Comment:
It'd be nice to upstream your patch, also to reduce our maintenance burden.  Building the shared library can be useful also for others.